### DISCUSSION TITLE
Are there issue with cognition.run lately?

### DISCUSSION THREAD

**Discussion by t-09** [jspsych/jsPsych#2349] [Q&A]
Hi everybody!
I'm not sure whether this is the right place to write about this. In case, please feel free to remove this post.

I'm using JsPsych together with Cognition.run for months and everything has worked out perfectly until now!
 
Lately, however, I am having some problems with cognition.run (i.e. it is slow, sometimes images during the experiment are not shown and sometimes I also see the ERROR 502 BAD GATEWAY message).

Is it just a problem of mine or maybe is the site under maintenance?
I don't know if it can help but I use JsPsych v6.2

Cheers,
Tony

**Comment by mustafayukselaud** [comment]
I experienced similar problems in the last few days. Experiments are running slow, sometimes it became unresponsive etc. I was assuming that there were some maintenance stuff going on at first but it is still problematic.

**Comment by pjkohler** [comment]
I am having issues as well, images sometimes not loaded despite preload, etc.

**Comment by GEJ1** [comment]
Yes, I'm experiencing this kind of problems too.
